static/sass/styles.scss

```
@@ -420,6 +427,10 @@ Importing color and theme updates for the brand 1.5 refresh
 		}
 	}
 
+	thead {
+		background-color: #0a1931 !important;
+	}
```

```suggestion
	tr {
		overflow-x: hidden;

		a.anchor-link {
			position: absolute;
		}
	}
```
we do not want to have an extra column in the table because it would look odd, so we can use `position: absolute` to get around it. The background-color becomes unnecessary once the anchor link is absolutely positioned

---

Got it!